### PR TITLE
LPAL-1133 Add X-Ray to service-front 

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

[LPAL-1133](https://opgtransform.atlassian.net/browse/LPAL-1133)

## Approach

See ticket.

## Learning

n/a

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [X] I have added mandatory tags to terraformed resources, where possible
* [X] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [X] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

[LPAL-1133]: https://opgtransform.atlassian.net/browse/LPAL-1133?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ntse — COMMENTED** [review]
A definition on `telemetry_requests_sampled_fraction` will also need adding to terraform/environment/modules/environment/variables.tf

**Review by townxelliot — COMMENTED** [review]
Log packets which are thrown away because they are too big.

**Review by nickdavis2001 — APPROVED** [review]

File: shared/composer.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
         "symfony/validator": "^4.2"
     },
     "require-dev": {
+        "hamcrest/hamcrest-php": "^2.0",
```
[the remark this answers is not in the document]
Comment (townxelliot) [reply shared/composer.json:22]: > Check this is used somewhere.

It is.

**Comment by townxelliot** [comment]
> Log packets which are thrown away because they are too big.

Done

**Review by nickdavis2001 — APPROVED** [review]

**Pull request merged**